### DISCUSSION TITLE
Unable to connect to node node after master upgrade

### DISCUSSION THREAD

**Discussion by clcc2019** [k3s-io/k3s#9783] [Q&A]
**Environmental Info:**
K3s Version: 
v1.29.2

Node(s) CPU architecture, OS, and Version: 
Linux ds 6.5.0-3-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.5.8-1 (2023-10-22) x86_64 GNU/Linux

Cluster Configuration:

**Describe the bug:**
upgrade master version to v1.29.2.
After the update, all working nodes cannot connect and will recover after restarting the working nodes. The problem should be that the worker node iptables is denying all traffic from the outside
**Steps To Reproduce:**

- Installed K3s: 

**Expected behavior:**

**Actual behavior:**

**Additional context / logs:**

**Comment by brandond** [comment]
> The problem should be that the worker node iptables is denying all traffic from the outside

I'm not aware of any rules that k3s would create that would block access to the node. You might use `iptables-save` to dump the generated rules, and see where the rules that you believe are blocking access are coming from.
